src/ruleset.xml

```diff
@@ -22,5 +22,6 @@
     <rule ref="./../vendor/magento/magento-coding-standard/Magento2" >
         <exclude name="Magento2.Annotation.MethodAnnotationStructure" />
         <exclude name="Magento2.Commenting.ClassPropertyPHPDocFormatting.Missing" />
+        <exclude name="Magento2.Annotation.MethodArguments.NoCommentBlock" />
```

Comment:
PhpCS requires docBlock even it absolutely useless. Examples:

![image](https://user-images.githubusercontent.com/127951620/236450498-62a37783-a0eb-4535-b065-f74e428fbb24.png)

![image](https://user-images.githubusercontent.com/127951620/236450576-48bb7619-053b-476b-bb71-0e75f02abc3a.png)